hfst-language-module/speller/index.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hfstspeller dtdversion="1.0" hfstversion="3">
+  <info>
+    <locale>{{languageCode}}</locale>
+    <!-- TODO: locale may need to be changed to 2-letter ISO 639-1 code -->
+    <title>Apertium morphological analyser-based speller ({{languageName}})</title>
+    <description>
+      A spellchecker for {{languageName}}.
+    </description>
+    <version vcsrev="GT_REVISION">GT_VERSION</version>
+    <date>DATE</date>
+    <producer>Apertium contributors</producer>
+    <contact email="feedback@divvun.no"
+        website="http://divvun.no"/>
```

Comment:
Upon review, maybe those 2 should be changed. And GT_VERSION isn't a thing for us - should be `@APERTIUM_AUTO_VERSION@`